baseapp/baseapp.go

```diff
@@ -29,6 +31,9 @@ type (
 	// Enum mode for app.runTx
 	runTxMode uint8
 
+	// Enum mode for app.runTx and app.runMsgs
+	MsgExecutionGuaranteeMode uint8
```

Comment:
nit: There's another place in which we call this "Atomicity", we might want to decide on a single name

Reply:
good call! I'll change this to `MsgExecutionAtomicity`